Sources/SwiftParser/Patterns.swift

```diff
@@ -70,12 +70,26 @@ extension Parser {
       )
     case (.lhs(.identifier), let handle)?:
       let identifier = self.eat(handle)
-      return RawPatternSyntax(
-        RawIdentifierPatternSyntax(
-          identifier: identifier,
-          arena: self.arena
+      if identifier.tokenText.isEditorPlaceholder {
+        return RawPatternSyntax(
+          RawExpressionPatternSyntax(
+            expression: RawExprSyntax(
+              RawEditorPlaceholderExprSyntax(
+                placeholder: identifier,
```

Comment:
The thing here is that the `identifier` includes `<#` and `#>`. 
Not sure how to come around that

Reply:
For now, that’s intentional. I don’t think that splitting the token into `<#`, the contents and `#>` provides too much value for now.